x/ibc/04-channel/keeper/handshake.go

```diff
@@ -296,17 +307,15 @@ func (k Keeper) ChanCloseInit(
 	ctx sdk.Context,
 	portID,
 	channelID string,
+	portCap capability.Capability,
 ) error {
-	// TODO: blocked by #5542
-	// capkey, found := k.GetChannelCapability(ctx, portID, channelID)
-	// if !found {
-	// 	return sdkerrors.Wrap(types.ErrChannelCapabilityNotFound, channelID)
-	// }
+	if _, ok := k.GetChannelCapability(ctx, portID, channelID); !ok {
```

Comment:
This should be turned into an `authenticateCapability` call, see [the spec](https://github.com/cosmos/ics/tree/cwgoes/capability-rework/spec/ics-004-channel-and-packet-semantics#opening-handshake):

```typescript
abortTransactionUnless(authenticateCapability(channelCapabilityPath(portIdentifier, channelIdentifier), capability))
```